xclim/ensembles/_base.py

```diff
@@ -105,6 +107,8 @@ def ensemble_mean_std_max_min(ens: xr.Dataset) -> xr.Dataset:
     ----------
     ens: xr.Dataset
       Ensemble dataset (see xclim.ensembles.create_ensemble).
+    weights: xr.DataArray
+      1-D array of weights following the 'realization' dimension. This array cannot contain missing values.
```

Comment:
Is this "1-D" requirement strict? Couldn't `weights` have any dimensions, as long as it's a subset of those of  `ens`?

Reply:
I haven't tried it. Weights alongside `realization` are needed, but I don't know how it would process weights alongside a second dimension. It would probably just ignore them.

Reply:
I think it would simply use the different realization weights for each coordinate of the other dimensions.

Reply:
You're right, I just tested it. I'll correct the Docstrings accordingly.